pkg/api/postage_test.go

```diff
@@ -232,6 +232,51 @@ func TestPostageGetStamps(t *testing.T) {
 	})
 }
 
+func TestPostageGetNonExistingStamps(t *testing.T) {
```

Comment:
This test is wrong. You are not creating a stampIssuer here, so there is nothing to iterate over. Also the name of this test is doesnt make sense.

It would be better to have just 1 test and have 2 subtests as in both the tests you are doing the same thing.